pkg/storage/storage.go

```diff
@@ -181,3 +184,48 @@ func parseCommit(r io.Reader, hash string) (Commit, error) {
 
 	return c, nil
 }
+
+//TreeEntry is a file or folder at a given path in a repository
+type TreeEntry struct {
+	Mode   string
+	Type   string
```

Comment:
This should probably be an "Enum" :thinking:

Reply:
Are you talking about iota? AFAIK there's only blob and tree, right? Maybe constants also work. But right now I don't really see it as a problem as it's never written anywhere, after the initial parsing from git's output.